app/components/signatures/publication-status.js

```diff
@@ -0,0 +1,28 @@
+import Component from '@glimmer/component';
+import {tracked} from "@glimmer/tracking";
+import { action } from "@ember/object";
+
+
+export default class SignaturesPublicationStatus extends Component {
+  @tracked signedResources = [];
+  @tracked publishedResource
+  @tracked ready = false;
+  @action  
+  async loadResource() {
+    console.log(this.args.versionedResource.get('signedResources'));
+    this.signedResources = await this.args.versionedResource.get('signedResources');
```

Comment:
you are overwriting the empty array here with something that is potentially null or undefined(if there are no signedResources), and then your publicationStatus logic crashes cause you are trying to access the length of it.
so either add a `|| []` here or change the logic in the getter